pandas/core/arrays/arrow/dtype.py

```diff
@@ -8,28 +8,42 @@
 
 from pandas.core.dtypes.base import StorageExtensionDtype
 
-from pandas.core.arrays.arrow import ArrowExtensionArray
-
 
 class ArrowDtype(StorageExtensionDtype):
     """
-    Base class for dtypes for BaseArrowArray subclasses.
+    Base class for dtypes for ArrowExtensionArray.
     Modeled after BaseMaskedDtype
     """
 
-    name: str
-    base = None
-    type: pa.DataType
-
     na_value = pa.NA
 
-    def __init__(self, storage="pyarrow") -> None:
-        super().__init__(storage)
+    def __init__(self, pa_dtype: pa.DataType) -> None:
+        super().__init__("pyarrow")
+        if not isinstance(pa_dtype, pa.DataType):
+            raise ValueError(
+                f"pa_dtype ({pa_dtype}) must be an instance "
+                f"of a pyarrow.DataType. Got {type(pa_dtype)} instead."
+            )
+        self.pa_dtype = pa_dtype
```

Comment:
Since this would be a public attribute, maybe call it `pyarrow_dtype` to be more descriptive?